fedWireMessage.go

```diff
@@ -131,6 +131,8 @@ type FEDWireMessage struct {
 	RemittanceFreeText *RemittanceFreeText `json:"remittanceFreeText,omitempty"`
 	// ServiceMessage
 	ServiceMessage *ServiceMessage `json:"serviceMessage,omitempty"`
+	// IsFedLineAdvantageCustomer
+	IsFedLineAdvantageCustomer bool
```

Comment:
I wonder if it'd be more effective to simply make IMAD an optional field, as opposed to introducing this boolean flag. My concern is that this change won't account for using the API to upload/validate a file (file upload, not JSON).

Another approach that would really enhance the library's flexibility would be the introduction of configurable validation. An example of this can be seen in our ACH library: https://github.com/moov-io/ach/blob/04b60863f3783485407844d17420a6b5d2f9ded4/file.go#L621

In Wire's case, the validateOpts would be added to the Wire file: https://github.com/moov-io/wire/blob/c5bf5440a57b9de5852d88a6f0c264e73c308034/file.go#L14

Thoughts?

Reply:
Agreed I think that's a cleaner approach. I'll make the adjustment

Reply:
Thanks! I really appreciate the contribution.

Reply:
I assume there would be a lot of changes when someone is a fedline advantage member.

Reply:
Actually the only change I'm aware is that the IMAD field is optional. I believe thats the only effect it has on the content of the file. Let me know if you're aware if any other changes relating to fedline advantage members.

Reply:
@atonks2 can you please review the new code. You mentioned support for uploading files. Are there other areas of the code where `ValidateOpts` needs to be implemented?